### ISSUE TITLE
Group 'Similar' Stocks

### ISSUE THREAD

**Issue by aaronlam88** [aaronlam88/cmpe295#130]
Block by Normalized Data.

After data is normalized, we should be able to 'group' stocks together.
We should only care about Close Price to make it  easier. Stocks that are always Close '+' should group together, and Close '-' group together. 
I still don't know how many group we will have and how to do this.  We may use this http://scikit-learn.org/stable/modules/generated/sklearn.cluster.MeanShift.html#sklearn.cluster.MeanShift
